Source/WebCore/xml/parser/XMLDocumentParserLibxml2.cpp

```diff
@@ -1113,15 +1113,6 @@ static void warningHandler(void* closure, const char* message, ...)
     va_end(args);
 }
 
-WTF_ATTRIBUTE_PRINTF(2, 3)
-static void fatalErrorHandler(void* closure, const char* message, ...)
-{
-    va_list args;
-    va_start(args, message);
-    getParser(closure)->error(XMLErrors::fatal, message, args);
```

Comment:
Can we get rid of `XMLErrors::fatal` from `Source/WebCore/xml/XMLErrors.h` if this was the only place it was used?

Reply:
I looked into Chromium source and noticed that they still maintain 'fatal' type errors in `XMLErrors.h` and `XMLErrors.cpp`.

Reply:
But where are the `fatal` type errors used?

Reply:
I just looked into it - https://gnome.pages.gitlab.gnome.org/libxml2/devhelp/libxml2-xmlerror.html

It still has 'XML_ERR_FATAL' in 'xmlErrorLevel'. So I think we cannot remove it.